website/profile/views.py

```diff
@@ -524,6 +528,10 @@ def impute_names(**kwargs):
     return auth_utils.impute_names(name)
 
 
+def help_mails(user, subscribe):
```

Comment:
Please give this function a more descriptive name, like `update_help_mails_subscription`